test/camunda-cloud/features/popup-menu/ReplaceMenuProviderSpec.js

```diff
@@ -66,12 +66,14 @@ describe('camunda-cloud/features - PopupMenu', function() {
 
       const startEventEntry = queryEntry(popupMenu, 'replace-with-none-start'),
             intermediateEventEntry = queryEntry(popupMenu, 'replace-with-none-intermediate-throw'),
-            errorEndEventEntry = queryEntry(popupMenu, 'replace-with-error-end');
+            errorEndEventEntry = queryEntry(popupMenu, 'replace-with-error-end'),
+            messageEndEventEntry = queryEntry(popupMenu, 'replace-with-message-end');
```

Comment:
Let's also add a new test case to check whether we can replace from a messageThrow and a messageEnd event to the repsective other events. E.g.;
```
    it('should contain options for MessageThrowEvent', ...
```

and

```
    it('should contain options for MessageEndEvent', ...
```